velox/vector/VectorStream.h

```diff
@@ -71,10 +71,12 @@ class VectorSerde {
       const Options* options = nullptr) = 0;
 };
 
-bool registerVectorSerde(std::unique_ptr<VectorSerde> serde);
+bool registerVectorSerde(std::unique_ptr<VectorSerde> serdeToRegister);
 
 bool isRegisteredVectorSerde();
 
+VectorSerde* getVectorSerde();
```

Comment:
I noticed there is one in velox/functions/prestosql/aggregates/MaxSizeForStatsAggregate.cpp. Do we want to consolidate?

Reply:
I think this aggregate function might work in an environment where VectorSerde is not registered (IDK maybe XStream or some other use cases where there is no need for shuffling). That is why they create/"register" their own serde there in the cpp